project_euler/Problem 5/solution1.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <math.h>
+/*
+Problem 4: 
+2520 is the smallest number that can be divided by each of the numbers from 1 to 10 without any remainder.
+What is the smallest positive number that is evenly divisible(divisible with no remainder) by all of the numbers from 1 to N?
+*/
+int isprime(int no) {
+	int sq;
+
+	if (no == 2) {
+		return 1;
+	}
+	else if (no%2 == 0) {
+		return 0;
+	}
+	sq = ((int)(sqrt(no))) + 1;
+	for (int i = 3; i < sq; i += 2) {
+		if (no%i == 0) {
+			return 0;
+		}
+	}
+	return 1;
+}
+int countExp(int number,int power){
+	int x=0;
+	while (1){
+		if (number%power==0){
+			number/=power;
+			x++;
+		}
+		else{return x;
+		}
+		
+	}
+}
+int main(){
+	int smallest=1,x,number,last=0;
+	scanf("%d",&number);
+	
+	int primes[100];
+	int powerList[100];
+	for (int i=0;i<100;i++){
```

Comment:
Here you can use [memset](https://www.tutorialspoint.com/c_standard_library/c_function_memset.htm)